nebula/ui/components/VPNConnectionInfoContent.qml

```diff
@@ -120,6 +120,7 @@ VPNFlickable {
 
                         VPNIcon {
                             id: arrowIcon
+                            anchors.horizontalCenter: parent.horizontalCenter
```

Comment:
using anchors inside a component that is in a layout will usually cause issues and unintended affects (and of course a qml warning 😂)

<img width="472" alt="Screen Shot 2023-01-11 at 4 28 19 PM" src="https://user-images.githubusercontent.com/15353801/211922703-34b70fb7-3d2c-47db-975f-3400137d731c.png">

as far as how to resolve this... it sort of depends on what the intended design is. 

- Do we want the arrow to always be dead center, splitting the two labels 50/50?
- Do we want the arrow to move in case we have a longer city on one side than the other?
- If we truncate one city, do we always want to truncate the other (for consistency) or do we want to try to always have at least one of the cities not truncated

@peterbenvenuto any thoughts?

Reply:
Good point, I played around with some of the longest names I could find (NYC->NYC) and not seeing the same issue you mentioned. I wonder why in my case it adds "..." and in your it does not? 🤔 

I suppose I'm leaning towards not centering the arrow but let's see what @peterbenvenuto thinks.

![Screenshot 2023-01-11 at 1 42 10 PM](https://user-images.githubusercontent.com/3746552/211924195-ced8c21d-fa60-4a8e-8e97-8b3783a4eae2.png)